### PR TITLE
Rename ossec.conf to manager.conf and agent.conf in Documentation

### PR DESCRIPTION
**Pull request by cborla**
|Related issue|
|---|
|#3579|

## Description
Rename ossec.conf to manager.conf and agent.conf in Documentation

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

## Note to the reviewer

This PR includes changes to the `redirect.js` script that need to be included in all production branches.

### CODE REVIEW

File: source/migrating-from-ossec/ossec-agent.rst [hunk 1]
```diff
@@ -74,12 +74,12 @@ Now we will restore the following files:
 
 .. code-block:: bash
 
-   $ cp -p /var/ossec_backup/etc/ossec.conf /var/ossec/etc/ossec.conf.orig
+   $ cp -p /var/ossec_backup/etc/agent.conf /var/ossec/etc/agent.conf.orig
```
Comment (TomasTurina) [inline source/migrating-from-ossec/ossec-agent.rst:77]:
```suggestion
   $ cp -p /var/ossec_backup/etc/ossec.conf /var/ossec/etc/ossec.conf.orig
```

File: source/migrating-from-ossec/ossec-server.rst [hunk 2]
```diff
@@ -74,14 +74,14 @@ Now we will restore the following files:
...
+   $ cp -p /var/ossec_backup/etc/manager.conf /var/ossec/etc/manager.conf.orig
    $ cp -p /var/ossec_backup/etc/local_internal_options.conf /var/ossec/etc/local_internal_options.conf
    $ cp -p /var/ossec_backup/etc/local_decoder.xml /var/ossec/etc/decoders/local_decoder.xml
    $ cp -p /var/ossec_backup/etc/shared/agent.conf /var/ossec/etc/shared/default/agent.conf
    $ cp -p /var/ossec_backup/rules/local_rules.xml /var/ossec/etc/rules/local_rules.xml
    $ cp -p /var/ossec_backup/queue/rids/sender_counter /var/ossec/queue/rids/sender_counter
 
-There have been some syntax changes, and new settings, incorporated to ``ossec.conf`` file. Please review this file manually in order to import your previous configuration. In addition, note note that ``agent.conf`` file directory has now changed to ``/var/ossec/etc/shared/default``.
+There have been some syntax changes, and new settings, incorporated to ``wazuh.conf`` file. Please review this file manually in order to import your previous configuration. In addition, note note that ``agent.conf`` file directory has now changed to ``/var/ossec/etc/shared/default``.
```
Comment (TomasTurina) [inline source/migrating-from-ossec/ossec-server.rst:84]:
```suggestion
There have been some syntax changes, and new settings, incorporated to ``manager.conf`` file. Please review this file manually in order to import your previous configuration. In addition, note note that ``agent.conf`` file directory has now changed to ``/var/ossec/etc/shared/default``.
```

File: source/user-manual/capabilities/policy-monitoring/openscap/oscap-configuration.rst [hunk 3]
```diff
@@ -41,7 +41,7 @@ This section describes how to evaluate the Payment Card Industry Data Security S
 
 Each agent must be properly identified in order to know which policy and profile to execute. To do this, configure ``<config-profile>`` with the desired identifier.
 
-Modify the ``ossec.conf`` file in the agent side to apply the desired profile:
+Modify the ``agent.conf`` file in the agent side to apply the desired profile:
```
Comment (javabalm) [inline source/user-manual/capabilities/policy-monitoring/openscap/oscap-configuration.rst:44]: Redundant information "agent side"

File: source/user-manual/capabilities/policy-monitoring/openscap/oscap-configuration.rst [hunk 4]
```diff
@@ -129,7 +129,7 @@ The Red Hat Security Response Team provides OVAL definitions for all vulnerabili
...
```
Comment (javabalm) [inline source/user-manual/capabilities/policy-monitoring/openscap/oscap-configuration.rst:132]: Redundant information "agent side"

File: source/user-manual/reference/ossec-conf/active-response.rst [hunk 5]
```diff
@@ -161,7 +161,7 @@ Sets timeouts in minutes for repeat offenders. This is a comma-separated list of
 +--------------------+-----------------------------+
 
 .. warning::
-    This option must be configured directly in the **ossec.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
+    This option must be configured directly in the **agent.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
 
 Sample Configuration
```
Comment (javabalm) [inline source/user-manual/reference/ossec-conf/active-response.rst:166]: See if it is better to divide this code block into agent.conf and manager.conf

File: source/user-manual/reference/ossec-conf/syscheck.rst [hunk 6]
```diff
@@ -468,7 +468,7 @@ Example:
 
 .. note::
 
-  This option is ignored when defined at *agent.conf* if ``allow_remote_prefilter_cmd`` is set to ``no`` at *ossec.conf*.
+  This option is ignored when defined at *agent.conf* if ``allow_remote_prefilter_cmd`` is set to ``no`` at *manager.conf*.
```
Comment (TomasTurina) [inline source/user-manual/reference/ossec-conf/syscheck.rst:471]:
```suggestion
  This option is ignored when defined at *shared/agent.conf* if ``allow_remote_prefilter_cmd`` is set to ``no`` at *agent.conf*.
```

File: source/user-manual/reference/ossec-conf/labels.rst [hunk 7]
```diff
@@ -12,7 +12,7 @@ labels
 	    <labels>
 	    </labels>
 
-The labels section of ``ossec.conf`` allows additional user-defined information about agents to be included in alerts. When email notifications are enabled, this additional data is also contained in the email alerts without any further configuration.
+The labels section of ``agent.conf`` allows additional user-defined information about agents to be included in alerts. When email notifications are enabled, this additional data is also contained in the email alerts without any further configuration.
```
Comment (TomasTurina) [inline source/user-manual/reference/ossec-conf/labels.rst:15]: Labels section is also available in `manager.conf`

File: source/user-manual/reference/ossec-conf/active-response.rst [hunk 8]
```diff
@@ -161,7 +161,7 @@ Sets timeouts in minutes for repeat offenders. This is a comma-separated list of
 +--------------------+-----------------------------+
 
 .. warning::
-    This option must be configured directly in the **ossec.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
+    This option must be configured directly in the **agent.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
```
Comment (TomasTurina) [inline source/user-manual/reference/ossec-conf/active-response.rst:164]:
```suggestion
    This option must be configured directly in the **agent.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **shared/agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
```

File: source/user-manual/reference/ossec-conf/index_agent.rst [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+            -->
+        </active-response>
+    </wazuh_config>
+
+The ``shared/agent.conf`` file is very similar to ``agent.conf``, but ``shared/agent.conf`` is used to centrally distribute configuration information to agents. See more :doc:`here <../centralized-configuration>`.
+
+Wazuh can be installed in two ways: as a manager by using the "server/manager" installation type and as an agent by using the "agent" installation type.
+
++---------------------------------------------------------------------+------------------------+
+| Configuration sections                                              | Supported installations|
```
Comment (TomasTurina) [inline source/user-manual/reference/ossec-conf/index_agent.rst:32]: I think the second column is not necessary.

File: source/user-manual/reference/ossec-conf/index_manager.rst [hunk 10]
```diff
@@ -1,13 +1,16 @@
...
 
-The ``ossec.conf`` file is the main configuration file on the Wazuh manager and it also plays an important role on the agents. It is located at ``/var/ossec/etc/ossec.conf`` both in the manager and agent on Linux machines. On Windows agents, we can find it at ``C:\Program Files (x86)\ossec-agent\ossec.conf``. It is recommended to back up this file before making changes on it. A configuration error may prevent Wazuh services from starting up.
+.. warning::
+    Upgrade from older version than v5.0.0, will create a new manager.conf file and current configuration will be saved into ossec.conf.backup file, restarting configuration from zero.
 
-The ``ossec.conf`` file is in XML format and all of its configuration options are nested in their appropriate section of the file. In this file, the outermost XML tag is ``<wazuh_config>``. There can be more than one ``<wazuh_config>`` tag.
+The ``manager.conf`` file is the main configuration file on the Wazuh manager. It is located at ``/var/ossec/etc/manager.conf`` on Linux machines.. It is recommended to back up this file before making changes on it. A configuration error may prevent Wazuh services from starting up.
```
Comment (TomasTurina) [inline source/user-manual/reference/ossec-conf/index_manager.rst:11]:
```suggestion
The ``manager.conf`` file is the main configuration file on the Wazuh manager. It is located at ``/var/ossec/etc/manager.conf`` on Linux machines. It is recommended to back up this file before making changes on it. A configuration error may prevent Wazuh services from starting up.
```

File: source/user-manual/reference/ossec-conf/index_manager.rst [hunk 11]
```diff
@@ -21,27 +24,23 @@ Here is an example of the proper location of the *alerts* configuration section:
         </alerts>
     </wazuh_config>
 
-The ``agent.conf`` file is very similar to ``ossec.conf`` but ``agent.conf`` is used to centrally distribute configuration information to agents. See more :doc:`here <../centralized-configuration>`.
+The ``shared/agent.conf`` file is very similar to ``manager.conf`` but ``shared/agent.conf`` is used to centrally distribute configuration information to agents. See more :doc:`here <../centralized-configuration>`.
```
Comment (TomasTurina) [inline source/user-manual/reference/ossec-conf/index_manager.rst:27]: Remove this.

File: source/user-manual/reference/ossec-conf/index_manager.rst [hunk 12]
```diff
@@ -21,27 +24,23 @@ Here is an example of the proper location of the *alerts* configuration section:
...
 
 Wazuh can be installed in two ways: as a manager by using the "server/manager" installation type and as an agent by using the "agent" installation type.
 
 +---------------------------------------------------------------------+------------------------+
 | Configuration sections                                              | Supported installations|
```
Comment (TomasTurina) [inline source/user-manual/reference/ossec-conf/index_manager.rst:32]: I think the second column is not necessary.

**Review by TomasTurina — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/ossec-conf/active-response.rst (hunk 8 above)
Comment (cborla) [reply source/user-manual/reference/ossec-conf/active-response.rst:164]: could be something like?

_This option must be configured in the **agent.conf** file in the agent, and also in the **manager.conf** file in manager side, even when using a manager/agent setup with centralized configuration of other settings via **shared/agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file._

File: source/user-manual/reference/ossec-conf/labels.rst (hunk 7 above)
Comment (cborla) [reply source/user-manual/reference/ossec-conf/labels.rst:15]: I guess this is valid:
_The labels section of ``agent.conf`` or ``manager.conf``, allows additional user-defined information about agents to be included in alerts. When email notifications are enabled, this additional data is also contained in the email alerts without any further configuration._

File: source/user-manual/capabilities/policy-monitoring/openscap/oscap-configuration.rst (hunk 3 above)
Comment (cborla) [reply source/user-manual/capabilities/policy-monitoring/openscap/oscap-configuration.rst]: Done

File: source/user-manual/capabilities/policy-monitoring/openscap/oscap-configuration.rst (hunk 4 above)
Comment (cborla) [reply source/user-manual/capabilities/policy-monitoring/openscap/oscap-configuration.rst]: Done

File: source/user-manual/capabilities/labels.rst [hunk 13]
```diff
@@ -76,7 +76,7 @@ When an alert is fired for an agent with the above configuration applied from th
     network.ip: 172.17.0.0
     network.mac: 02:42:ac:11:00:02
     Rule: 550 (level 7) -> 'Integrity checksum changed.'
-    Integrity checksum changed for: '/var/ossec/etc/ossec.conf'
+    Integrity checksum changed for: '/var/ossec/etc/manager.conf'
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/capabilities/labels.rst]: done

File: source/user-manual/capabilities/log-data-collection/log-data-configuration.rst [hunk 14]
```diff
@@ -15,7 +15,7 @@ Configuration
 Basic usage
 -----------
 
-Log data collection is configured in the :ref:`ossec.conf <reference_ossec_conf>` file primarily in the :ref:`localfile <reference_ossec_localfile>`, :ref:`remote <reference_ossec_remote>` and :ref:`global <reference_ossec_global>` sections. Configuration of log data collection can also be completed in the :ref:`agent.conf <reference_agent_conf>` file to centralize the distribution of these configuration settings to relevant agents.
+Log data collection is configured in the :ref:`agent.conf <reference_agent_conf>` file primarily in the :ref:`localfile <reference_ossec_localfile>`, :ref:`remote <reference_ossec_remote>` and :ref:`global <reference_ossec_global>` sections. Configuration of log data collection can also be completed in the :ref:`agent.conf <reference_agent_conf>` file to centralize the distribution of these configuration settings to relevant agents.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/capabilities/log-data-collection/log-data-configuration.rst]: done

File: source/user-manual/capabilities/policy-monitoring/rootcheck/rootcheck-configuration.rst [hunk 15]
```diff
@@ -10,7 +10,7 @@ Configuration
 Basic usage
 -----------
 
-To configure the options for rootcheck, go to the :ref:`Rootcheck section <reference_ossec_rootcheck>` in :ref:`ossec.conf <reference_ossec_conf>`. The most common configuration options are: :ref:`frequency <reference_ossec_rootcheck_frequency>` and :ref:`system-audit <reference_ossec_rootcheck_audit>`
+To configure the options for rootcheck, go to the :ref:`Rootcheck section <reference_ossec_rootcheck>` in :ref:`agent.conf <reference_agent_conf>`. The most common configuration options are: :ref:`frequency <reference_ossec_rootcheck_frequency>` and :ref:`system-audit <reference_ossec_rootcheck_audit>`
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/capabilities/policy-monitoring/rootcheck/rootcheck-configuration.rst]: done

File: source/user-manual/kibana-app/features/ruleset.rst [hunk 16]
```diff
@@ -15,7 +15,7 @@ A special search syntax can be used on the bar for filtering rules and decoders,
   :align: center
   :width: 100%
 
-When the user clicks on a rule or decoder, it will open a detail view, where you can find the complete information. In the different sections you can find :ref:`PCI DSS <pci_dss>` or :ref:`GDPR <gdpr>` requirements, :ref:`groups <reference_agent_conf>`, regular expressions, and other details. The user can click on some elements to activate a filter on the previous list.
+When the user clicks on a rule or decoder, it will open a detail view, where you can find the complete information. In the different sections you can find :ref:`PCI DSS <pci_dss>` or :ref:`GDPR <gdpr>` requirements, :ref:`groups <reference_client_conf>`, regular expressions, and other details. The user can click on some elements to activate a filter on the previous list.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/kibana-app/features/ruleset.rst]: done

File: source/user-manual/reference/daemons/ossec-execd.rst [hunk 17]
```diff
@@ -10,7 +10,8 @@ The ossec-execd program runs active responses by initiating the configured scrip
 +-----------------+-------------------------------------------------------------------------------------------------+
 | **-c <config>** | Run using <config> as the configuration file.                                                   |
 +                 +-------------------------------------------+-----------------------------------------------------+
-|                 | Default value                             | /var/ossec/etc/ossec.conf                           |
+|                 | Default value                             | /var/ossec/etc/manager.conf                         |
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/reference/daemons/ossec-execd.rst]: done

File: source/user-manual/reference/daemons/ossec-syscheckd.rst [hunk 18]
```diff
@@ -5,12 +5,12 @@
...
 ===============
 
-The ossec-syscheckd program checks configured files for changes to the checksums, permissions and ownership.  It is run using ossec-control.
+The ossec-syscheckd program checks configured files for changes to the checksums, permissions and ownership. It is run using ossec-control.
 
 +-----------------+-------------------------------------------------------------------------------------------------+
 | **-c <config>** | Run using <config> as the configuration file.                                                   |
 +                 +-------------------------------------------+-----------------------------------------------------+
-|                 | Default value                             | /var/ossec/etc/ossec.conf                           |
+|                 | Default value                             | /var/ossec/etc/manager.conf                         |
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/reference/daemons/ossec-syscheckd.rst]: done

File: source/user-manual/reference/ossec-conf/syscheck.rst (hunk 6 above)
Comment (cborla) [reply source/user-manual/reference/ossec-conf/syscheck.rst]: done

File: source/user-manual/reference/ossec-conf/index_agent.rst (hunk 9 above)
Comment (cborla) [reply source/user-manual/reference/ossec-conf/index_agent.rst]: Done

File: source/user-manual/reference/ossec-conf/index_manager.rst (hunk 10 above)
Comment (cborla) [reply source/user-manual/reference/ossec-conf/index_manager.rst]: done

File: source/user-manual/reference/ossec-conf/index_manager.rst (hunk 11 above)
Comment (cborla) [reply source/user-manual/reference/ossec-conf/index_manager.rst]: done

File: source/user-manual/reference/ossec-conf/index_manager.rst (hunk 12 above)
Comment (cborla) [reply source/user-manual/reference/ossec-conf/index_manager.rst]: Done

**Review by TomasTurina — CHANGES_REQUESTED** [review]

File: source/_static/js/redirects.js [hunk 19]
```diff
@@ -34,6 +34,21 @@ removedUrls['x.y'] = [
 ];
 */
 
+newUrls['5.0'] = [
+  '/user-manual/reference/ossec-conf/index_agent.html',
+  '/user-manual/reference/ossec-conf/index_manager.html',
+  '/user-manual/reference/ossec-conf/active-response_agent.html',
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/_static/js/redirects.js:40]: Done

File: source/user-manual/reference/ossec-conf/active-response_agent.rst [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+=======================
+
+.. topic:: XML section name
+
+	.. code-block:: xml
+
+		<active-response>
+		</active-response>
+
+In the active response configuration section, an existing command is bound to one or more rules or rule types along with additional criteria for when to execute the command. There is no limit to the number of active responses that can be used, however, each active response must be configured in its own separate ``<active-response>`` section.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/reference/ossec-conf/active-response_agent.rst]: Done

File: source/user-manual/reference/ossec-conf/active-response_agent.rst [hunk 21]
```diff
@@ -0,0 +1,64 @@
...
+
+.. note:: It is also necessary include active-response configuration in the manager side, check :ref:`here <reference_ossec_active_response_manager>`.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/reference/ossec-conf/active-response_agent.rst]: Done

File: source/user-manual/reference/ossec-conf/active-response_manager.rst [hunk 22]
```diff
@@ -14,14 +14,12 @@ active-response
 
 In the active response configuration section, an existing command is bound to one or more rules or rule types along with additional criteria for when to execute the command. There is no limit to the number of active responses that can be used, however, each active response must be configured in its own separate ``<active-response>`` section.
 
+.. note:: It is also necessary include active-response configuration in the agent side, check :ref:`here <reference_ossec_active_response_agent>`.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/reference/ossec-conf/active-response_manager.rst]: Done

File: source/user-manual/reference/ossec-conf/active-response_manager.rst [hunk 23]
```diff
@@ -149,26 +142,15 @@ Specifies how long in seconds before the reverse command is executed.  When ``re
...
-
-+--------------------+-----------------------------+
-| **Default value**  | n/a                         |
-+--------------------+-----------------------------+
-| **Allowed values** | A positive number (minutes) |
-+--------------------+-----------------------------+
-
 .. warning::
-    This option must be configured directly in the **ossec.conf** file of the agent, even when using a manager/agent setup with centralized configuration of other settings via **agent.conf**. Apart from that, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
+    This option must be configured in the **manager.conf** file, it has to be defined in the upper ``<active-response>`` section found in the configuration file.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/reference/ossec-conf/active-response_manager.rst]: Done

File: source/user-manual/reference/ossec-conf/labels_manager.rst [hunk 24]
```diff
@@ -0,0 +1,63 @@
...
+======
+
+.. topic:: XML section name
+
+	.. code-block:: xml
+
+	    <labels>
+	    </labels>
+
+The labels section of ``manager.conf``, allows additional user-defined information about agents to be included in alerts. When email notifications are enabled, this additional data is also contained in the email alerts without any further configuration.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/reference/ossec-conf/labels_manager.rst]: Done

File: source/deploying-with-ansible/reference.rst [hunk 25]
```diff
@@ -346,124 +346,9 @@ Wazuh Manager
         user: null
         showlogs: null
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/deploying-with-ansible/reference.rst:348]: Done

File: source/deploying-with-ansible/reference.rst [hunk 26]
```diff
@@ -346,124 +346,9 @@ Wazuh Manager
...
-      interval: '12h'
-      skip_nfs: 'yes'
-      day: ''
-      wday: ''
-      time: ''
-
 **wazuh_manager_vulnerability_detector**
 
-  Configures the :ref:`vulnerability-detector<vuln_detector>` section from ``ossec.conf``.
+  Configures the :ref:`vulnerability-detector<vuln_detector>` section from ``agent.conf``.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/deploying-with-ansible/reference.rst]: Done

File: source/user-manual/capabilities/active-response/how-it-works.rst [hunk 27]
```diff
@@ -73,31 +73,31 @@ Default Active response scripts
...
++--------------------------+-----------------------------------------------------------------+
+| ipfw                     | Firewall-drop response script created for ipfw                  |
++--------------------------+-----------------------------------------------------------------+
+| npf                      | Firewall-drop response script created for npf                   |
++--------------------------+-----------------------------------------------------------------+
+| wazuh-slack              | Posts modifications on Slack                                    |
++--------------------------+-----------------------------------------------------------------+
+| pf                       | Firewall-drop response script created for pf                    |
++--------------------------+-----------------------------------------------------------------+
+| restart-wazuh            | Automatically restarts Wazuh when manager.conf has been changed |
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/capabilities/active-response/how-it-works.rst]: Done

File: source/user-manual/capabilities/file-integrity/fim-configuration.rst [hunk 28]
```diff
@@ -21,7 +21,7 @@ After the installation, the Wazuh manager and the Wazuh agent have defined a :re
 #. `Configuring where the database is to be stored`_
 #. `Configuring synchronization`_
 
-Syscheck component is configured both in the Wazuh manager's and in the Wazuh agent's :ref:`ossec.conf <reference_ossec_conf>` file. This capability can be also configured remotely using centralized configuration and the :ref:`agent.conf <reference_agent_conf>` file.
+Syscheck component is configured both in the Wazuh manager's and in the Wazuh agent's :ref:`agent.conf <reference_client_conf>` file. This capability can be also configured remotely using centralized configuration and the :ref:`shared/agent.conf <reference_agent_conf>` file.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/user-manual/capabilities/file-integrity/fim-configuration.rst]: Done

**Review by TomasTurina — APPROVED** [review]
LGTM!

**Review by TomasTurina — APPROVED** [review]

File: source/user-manual/reference/ossec-conf/syscheck.rst [hunk 29]
```diff
@@ -1,6 +1,6 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 .. meta::
-  :description: The ossec.conf file is the main configuration file on the Wazuh manager and it also plays an important role on the agents. Learn more about it and check out an example here. 
+  :description: The manager.conf file is the main configuration file on the Wazuh manager and it also plays an important role on the agents. Learn more about it and check out an example here.
```
Comment (TomasTurina) [inline source/user-manual/reference/ossec-conf/syscheck.rst:3]: What is this description? Is this necessary?

File: source/learning-wazuh/suricata.rst [hunk 30]
```diff
@@ -232,7 +232,7 @@ the shared configuration with their local configuration.
 
         # service wazuh-manager restart
 
-   Each agent should pull down and apply this additional configuration almost immediately. You can find the fetched configuration on each agent at ``/var/ossec/etc/shared/agent.conf``.
+   Each agent should pull down and apply this additional configuration almost immediately. You can find the fetched configuration on each agent at ``/var/ossec/etc/shared/default/shared.conf``.
```
[the remark this answers is not in the document]
Comment (cborla) [reply source/learning-wazuh/suricata.rst]: Done

File: source/user-manual/reference/ossec-conf/syscheck.rst (hunk 29 above)
Comment (cborla) [reply source/user-manual/reference/ossec-conf/syscheck.rst]: Removed, done.

**Review by TomasTurina — APPROVED** [review]

File: source/_templates/installations/basic/wazuh/common/configure_wazuh_master_node.rst [hunk 31]
```diff
@@ -1,6 +1,6 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-Configure the cluster node by editing the following settings in ``/var/ossec/etc/ossec.conf``:
+Configure the cluster node by editing the following settings in ``/var/ossec/etc/manager.conf``:
```
Comment (MarVergara) [inline source/_templates/installations/basic/wazuh/common/configure_wazuh_master_node.rst:3]:
```suggestion
Configure the cluster node by editing the following settings in the ``/var/ossec/etc/manager.conf`` file:
```

**Review by MarVergara — COMMENTED** [review]
Submitting general editorial feedback without explicit approval. 
Editorial review is done but I still need assistance from the content team to review some file changes.

**Comment by MarVergara** [comment]
**Additional changes required to source/_static/js/redirects.js** 

- Add removed URLs  to 
```
`removedUrls['5.0'] = [`
];
```
-  Add URLs of files with changed names to `redirections.push`:

.../reference/ossec-conf/active-response.rst → ...ce/ossec-conf/active-response_manager.rst 

...al/reference/ossec-conf/agent-upgrade.rst → ...erence/ossec-conf/agent-upgrade_agent.rst 

 ...ser-manual/reference/ossec-conf/index.rst → ...al/reference/ossec-conf/index_manager.rst 

...er-manual/reference/ossec-conf/labels.rst → ...ual/reference/ossec-conf/labels_agent.rst 

...r-manual/reference/ossec-conf/logging.rst → .../reference/ossec-conf/logging_manager.rst

File: source/user-manual/reference/centralized-configuration.rst [hunk 32]
```diff
@@ -1,17 +1,14 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta::
-  :description: Learn how to remotely configure agents using agent.conf. In this section you will find which capabilities can be configured remotely. 
+.. _reference_shared_conf:
```
Comment (MarVergara) [inline source/user-manual/reference/centralized-configuration.rst:4]: Missing meta description. Please, add this element and make sure it corresponds with the content of the section. 
```suggestion


```

File: source/user-manual/reference/ossec-conf/syscheck.rst [hunk 33]
```diff
@@ -1,7 +1,4 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-.. meta::
-  :description: The ossec.conf file is the main configuration file on the Wazuh manager and it also plays an important role on the agents. Learn more about it and check out an example here. 
-
```
Comment (MarVergara) [inline source/user-manual/reference/ossec-conf/syscheck.rst:2]: Missing meta description. Please, add this element and make sure it corresponds with the content of the section.

```suggestion


```

**Review by MarVergara — APPROVED** [review]

**Review by Arcapas — APPROVED** [review]
LGTM

**Pull request merged**